apps/taxes/src/modules/taxes/tax-provider-utils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * The providers sdk types claim to sometimes return undefined.
```

Comment:
Improvement idea: maybe leveraging the Zod for data validation would be a good idea? It's already in the deps.
Let me know if you want to brainstom about this.

Reply:
agree, we can build Value Objects around Zod